content/Reducing_Power_Consumption.md

```diff
@@ -5,18 +5,19 @@ batteries.
 # Using the HALT Instruction
 
 The HALT instruction should be used whenever possible to reduce power
-consumption & extend the life of the batteries. This command stops the
+consumption and extend battery life. This instruction halts the
 system clock, reducing the power consumption of both the CPU and ROM.
```

Comment:
Actually, does it? The PPU and APU keep ticking, only the CPU starts sleeping; and I'm not sure if its clock actually stops... cc @Gekkio

Reply:
Waiting for @Gekkio

Reply:
I'm removing that statement, until we can't prove it.